app/models/health_care_application.rb

```diff
@@ -283,21 +287,23 @@ def log_submission_failure_details
   def send_failure_email
     if Flipper.enabled?(:hca_submission_failure_email_va_notify)
       begin
-        email = parsed_form['email']
+        first_name = parsed_form.dig('veteranFullName', 'first')
         template_id = Settings.vanotify.services.health_apps_1010.template_id.form1010_ez_failure_email
         api_key = Settings.vanotify.services.health_apps_1010.api_key
 
+        personalisations = first_name ? { 'salutation' => "Dear #{first_name}," } : nil
+
         VANotify::EmailJob.perform_async(
           email,
           template_id,
-          nil,
+          personalisations,
           api_key
         )
       rescue => e
         log_exception_to_sentry(e)
       end
     else
-      HCASubmissionFailureMailer.build(parsed_form['email'], google_analytics_client_id).deliver_now
+      HCASubmissionFailureMailer.build(email, google_analytics_client_id).deliver_now
```

Comment:
No idea. 🤷 This is going to go away with the new changes anyways and it will be using the sidekiq worker so it will be more similar to `deliver_later`.

Reply:
`deliver_later` queues the email to be sent async. `deliver_now` is sent immediately.